.github/CONTRIBUTING.md

```diff
@@ -1,65 +1,56 @@
+* Add a feature or change one by opening an issue describing your change. This gives the team and community a chance to give feedback before you spend time on something already underway,could be done differently, or should be avoided.
+* Stick to NewPipe's style conventions of [checkStyle](https://github.com/checkstyle/checkstyle). It runs each time you build the project.
+* Do not bring non-free software (e.g. binary blobs) into the project. Make sure you do not introduce Google
   libraries.
 * Stick to [F-Droid contribution guidelines](https://f-droid.org/wiki/page/Inclusion_Policy).
-* Make changes on a separate branch with a meaningful name, not on the master neither dev branch. This is commonly known as *feature branch workflow*. You
-  may then send your changes as a pull request (PR) on GitHub.
-* When submitting changes, you confirm that your code is licensed under the terms of the
-  [GNU General Public License v3](https://www.gnu.org/licenses/gpl-3.0.html).
-* Please test (compile and run) your code before you submit changes! Ideally, provide test feedback in the PR
+* Make changes on a separate branch with a meaningful name, not on the _master_ nor the _dev_ branch. This is commonly known as *feature branch workflow*. You may then send your changes as a pull request (PR) on GitHub.
+* Please test (compile and run) your code before submitting changes! Ideally, provide test feedback in the PR
   description. Untested code will **not** be merged!
-* Try to figure out yourself why builds on our CI fail.
-* Make sure your PR is up-to-date with the rest of the code. Often, a simple click on "Update branch" will do the job,
-  but if not, you are asked to rebase the dev branch manually and resolve the problems on your own. You can find help [on the wiki](https://github.com/TeamNewPipe/NewPipe/wiki/How-to-merge-a-PR). That will make the
+* Make sure your PR is up-to-date with the rest of the code. Often, a simple click on "Update branch" will do the job, but if not, you are asked to rebase the dev branch manually and resolve the problems on your own. You can find help [on the wiki](https://github.com/TeamNewPipe/NewPipe/wiki/How-to-merge-a-PR). That makes the
   maintainers' jobs way easier.
-* Please show intention to maintain your features and code after you contributed it. Unmaintained code is a hassle for
-  the core developers, and just adds work. If you do not intend to maintain features you contributed, please think again
-  about submission, or clearly state that in the description of your PR.
+* Please show intention to maintain your features and code after you contributed it. Unmaintained code is a hassle for core developers. If you do not intend to maintain features you contributed, please rethink your submission, or clearly state that in the PR description.
 * Respond yourselves if someone requests changes or otherwise raises issues about your PRs.
```

Comment:
```suggestion
* Respond yourself if someone requests changes or otherwise raises issues about your PRs.
```

Reply:
Already clear from "your".

Reply:
Sorry, I didn't get it.

Reply:
Since it says "your", there is no need to have "yourself/yourselves".

Reply:
Ah. I didn't see the changed wording then, so I was confused.